lms/djangoapps/teams/static/teams/js/views/teams_tab.js

```diff
@@ -255,6 +261,32 @@
                 },
 
                 /**
+                 *
+                 * The backbone router entry for editing team members, using topic and team IDs.
+                 */
+                editTeamMembers: function (topicID, teamID) {
+                    var self = this;
+                    this.getTopic(topicID).done(function (topic) {
```

Comment:
This could be cleaned up by using `$.when`, similarly to [`topic_teams.js`](https://github.com/edx/edx-platform/blob/master/lms/djangoapps/teams/static/teams/js/views/topic_teams.js#L24-L27). I think something like:

```javascript
$.when(this.getTopic(topicID), this.getTeam(teamID, true)).done(
    function (topic, team) {
        // ...
    }
);
```